### ISSUE TITLE
[Release][C#] Source verification for C# on 15.0.0 Release Candidate fails with `file does not exist: artifacts/Apache.Arrow/Release/netstandard1.3/Apache.Arrow.pdb`

### ISSUE THREAD

**Issue by raulcd** [apache/arrow#39598]
### Describe the bug, including details regarding any error messages, version, and platform.

I have been executing the source verification jobs both on the nightly jobs and the maintenance branch and I haven't found any prior issues but as soon as I've created the Release Candidate 0 for 15.0.0 all the C# verification jobs for the sources fail with the following error:
```
  Successfully created package '/tmp/arrow-15.0.0.NZGxf/apache-arrow-15.0.0/csharp//artifacts/Apache.Arrow.Flight.Sql/Release/Apache.Arrow.Flight.Sql.15.0.0.snupkg'.
file does not exist: artifacts/Apache.Arrow/Release/netstandard1.3/Apache.Arrow.pdb
Failed to verify release candidate. See /tmp/arrow-15.0.0.NZGxf for details.
```
The Release Candidate 0 jobs that are failing can be seen here:
https://github.com/apache/arrow/pull/39597#issuecomment-1890973621
and one example of failed job here: https://github.com/ursacomputing/crossbow/actions/runs/7519893060/job/20468954451

The maintenance branch successful jobs can be seen here:
https://github.com/apache/arrow/pull/39545#issuecomment-1889979579

### Component(s)

C#, Release

**Comment by raulcd** [comment]
@CurtHagenlocher @eerhardt any idea what might be going wrong on the C# verification?

**Comment by CurtHagenlocher** [comment]
As 1.3 has been obsolete for some time, I removed support for it. Am on a plane now but will try to understand the failure later today or tomorrow.Sent from my iPhoneOn Jan 14, 2024, at 7:06 AM, Raúl Cumplido ***@***.***> wrote:﻿[apache/arrow] [Release][C#] Source verification for C# on 15.0.0 Release Candidate fails with `file does not exist: artifacts/Apache.Arrow/Release/netstandard1.3/Apache.Arrow.pdb` (Issue #39598)Describe the bug, including details regarding any error messages, version, and platform.I have been executing the source verification jobs both on the nightly jobs and the maintenance branch and I haven't found any prior issues but as soon as I've created the Release Candidate 0 for 15.0.0 all the C# verification jobs for the sources fail with the following error:  Successfully created package '/tmp/arrow-15.0.0.NZGxf/apache-arrow-15.0.0/csharp//artifacts/Apache.Arrow.Flight.Sql/Release/Apache.Arrow.Flight.Sql.15.0.0.snupkg'.
file does not exist: artifacts/Apache.Arrow/Release/netstandard1.3/Apache.Arrow.pdb
Failed to verify release candidate. See /tmp/arrow-15.0.0.NZGxf for details.
The Release Candidate 0 jobs that are failing can be seen here:#39597 (comment)and one example of failed job here: https://github.com/ursacomputing/crossbow/actions/runs/7519893060/job/20468954451The maintenance branch successful jobs can be seen here:#39545 (comment)Component(s)C#, Release—Reply to this email directly, view it on GitHub or unsubscribe.You are receiving this email because you are subscribed to this thread.Triage notifications on the go with GitHub Mobile for iOS or Android.
